### PR TITLE
feat: minting subscription

### PR DESCRIPTION
**Pull request by aaitor**
## Description

Add a description of your changes here.

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] Documentation

### CODE REVIEW

File: src/utils/utils.ts [hunk 1]
```diff
@@ -66,6 +66,18 @@ export const loadNeverminedConfigContract = (config: ConfigEntry): Contract => {
   )
 }
 
+export const loadSubscriptionContract = (config: ConfigEntry, nftAddress: string): Contract => {
+  const abiNvmConfig = `${ARTIFACTS_PATH}/NFT721SubscriptionUpgradeable.${config.networkName?.toLowerCase()}.json`
+  console.log(abiNvmConfig)
```
Comment (eruizgar91) [inline src/utils/utils.ts]: Better use the logger than console.log

File: test/integration/NFTs721Subscription.test.ts [hunk 2]
```diff
@@ -149,4 +149,15 @@ describe('Subscription NFTs (ERC-721) e2e Testing', () => {
 
   })  
 
+  test('As NFT Publisher I can whitelist to an account', async () => {
+    
+    const whitelistCommand = `${baseCommands.nfts721.mintSubscription} "${nftAddress}" --receiver ${execOpts.accounts[2]} --tokenId 1`
+    console.debug(`COMMAND: ${whitelistCommand}`)
```
Comment (eruizgar91) [inline test/integration/NFTs721Subscription.test.ts:155]: Not sure how the test are logging this, but better use the logger if possible

File: src/commands/nfts/mintSubscriptionNft.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+  logger.debug(
+    chalk.dim(`Receiver: ${chalk.whiteBright(receiver)}`)
+  )
+
+  const tokenId = argv.tokenId && argv.tokenId.toString().length > 0 ? Number(argv.tokenId) : Number(0)
+  
+  const subscriptionContract = loadSubscriptionContract(config, nftAddress)
+  
+  console.log(await subscriptionContract.symbol())
```
Comment (eruizgar91) [inline src/commands/nfts/mintSubscriptionNft.ts]: use logger

**Review by eruizgar91 — CHANGES_REQUESTED** [review]

File: test/integration/NFTs721Subscription.test.ts (hunk 2 above)
Comment (aaitor) [reply test/integration/NFTs721Subscription.test.ts:155]: in the testing we are using console everywhere

**Review by eruizgar91 — APPROVED** [review]

**Pull request merged**